### DISCUSSION TITLE
Is events, tasks, free-busy info and notes supported?

### DISCUSSION THREAD

**Discussion by lamyergeier** [jelmer/xandikos#189] [Q&A] [answered]
> From [CalDAV - Wikipedia](https://en.wikipedia.org/wiki/CalDAV)
> 
> The architecture of CalDAV (partially inherited from the underlying specifications) organizes the data (events, tasks, free-busy info, notes) in directories (collections), where multiple items (resources) reside.

Are all of the data: events, tasks, free-busy info, notes; supported by xandikos. How can I sync tasks from Android to Xandikos. I use onlu open source tools: DAVx and Simple Calendar app on Android. At the moment my tasks are not getting synced to Thunderbird.

**Comment by jelmer** [comment, marked as answer]
Events, tasks and notes are all supported. Free-busy info can be stored as-is, but currently can't be generated for your existing events.

Syncing tasks can be done with DAVx; this is what I use myself.
